tests/contrib/patch.py

```diff
@@ -229,15 +254,16 @@ def test_patch_after_import(self):
 
                 import redis
                 ddtrace.patch(redis=True)
-                self.assert_patched(redis)
+                self.assert_module_patched(redis)
             """
-            for module_name, integration_name, _ in self.MODULES:
-                self.assert_module_not_imported(module_name)
-                module = importlib.import_module(module_name)
-                self.patch(**{integration_name: True})
-                self.assert_patched(module)
-
-        @require_modules
+            self.assert_module_not_imported(self.__module_name__)
+            module = importlib.import_module(self.__module_name__)
+            self.patch(**{self.__integration_name__: True})
+            self.assert_module_patched(module)
+            self.patch(**{self.__integration_name__: True})
+            self.assert_module_patched(module)
```

Comment:
Why do we patch twice?

Should this be `self.assert_not_double_patched(module)`?